### PR TITLE
add license, package control commands list, updated README

### PR DESCRIPTION
**Pull request by luttig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,15 +32,17 @@ git clone ssh://git@github.com/sourcegraph/sourcegraph-sublime
 
 ## Usage
 
-Sourcegraph for Sublime should automatically open a browser tab to initialize your Sourcegraph session when in Go files. If, for any reason, your tab gets closed, press <kbd>&#8984;</kbd>+<kbd>.</kbd>, <kbd>&#8984;</kbd>+<kbd>0</kbd> in any file to reopen the Sourcegraph tab. Alternatively, you can click on `Sublime Text > Preferences > Package Settings > Sourcegraph > Reopen Browser Tab`. As you navigate through Go files, your browser should automatically load the definition and references for recognized symbols.
+Sourcegraph for Sublime should automatically open a browser tab to initialize your Sourcegraph session when in Go files. If, for any reason, your tab gets closed, press <kbd>ctrl</kbd><kbd>alt</kbd><kbd>o</kbd> in any file to reopen the Sourcegraph tab. Alternatively, you can click on `Sublime Text > Preferences > Package Settings > Sourcegraph > Reopen Browser Tab`. As you navigate through Go files, your browser should automatically load the definition and references for recognized symbols.
```
Comment (wholien) [inline README.md]: Should not say "automatically open a browser tab" as `AUTO_OPEN` is now `false`

Comment (luttig) [inline README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -53,24 +55,24 @@ Sourcegraph for Sublime assumes a `GOBIN` of `/usr/local/bin/go`, and loads your
 
 ### Auto-process selection
 
-Sourcegraph automatically processes your code as you type. If you want to disable this feature, set the `AUTO_PROCESS` flag to `false`. If you disable it, you must press <kbd>&#8984;</kbd>+<kbd>,</kbd>, <kbd>&#8984;</kbd>+<kbd>0</kbd> to process your current selection.
+With the auto-process feature, Sourcegraph automatically processes your code as you type. If you want to toggle this feature, set the `AUTO_PROCESS` flag in your settings file. If you set it to `false`, you must press <kbd>ctrl</kbd><kbd>alt</kbd><kbd>j</kbd> to process your current selection.
```
Comment (wholien) [inline README.md]: `AUTO_PROCESS` is default set to false - should that be made more clear? The wording in the last sentence, "If you set it to `false`," gives the impression that the user has to turn it off themselves. We should be more explicit with the fact that it's default off/`false`. Maybve say "set the `AUTO_PROCESS` flag in your settings file to `true`?" From a first read through it is not very clear.

Comment (luttig) [inline README.md]: fixed in https://github.com/sourcegraph/sourcegraph-sublime/pull/17/commits/5a6b8e12f4a4a6ba6f75c989fd33a86f7cd180b7

File: README.md [hunk 3]
```diff
@@ -53,24 +55,24 @@ Sourcegraph for Sublime assumes a `GOBIN` of `/usr/local/bin/go`, and loads your
...
 ### Auto-open browser
 
-This setting auto-opens the channel in your browser when you click on a definition in Go. If it is set to false, then you can access the channel by clicking <kbd>&#8984;</kbd>+<kbd>.</kbd>, <kbd>&#8984;</kbd>+<kbd>0</kbd>. To disable auto-opening, set the `AUTO_OPEN` flag to `false` in your settings:
+If set to `true`, this setting will auto-open the Sourcegraph channel in your browser when you click on a definition in Go. If it is set to `false`, then you can access the channel by clicking <kbd>ctrl</kbd><kbd>alt</kbd><kbd>o</kbd>. To disable auto-opening, set the `AUTO_OPEN` flag to `false` in your settings:
 
 ```yml
 {
 	...
-	"AUTO_OPEN": false,
+	"AUTO_OPEN": true,
```
Comment (wholien) [inline README.md]: The last line in the description says "To disable auto-opening, set the `AUTO_OPEN` flag to `false` in your settings:" but the code snippet sets it to true - should it to "set ... to `true`"?

File: README.md [hunk 4]
```diff
@@ -79,10 +81,13 @@ This setting auto-opens the channel in your browser when you click on a definiti
 
 This setting gives verbose output from Sourcegraph for Sublime to the Sublime Text console, which can be helping when troubleshooting Sourcegraph for Sublime. To open the Sublime console, simply type <kbd>control</kbd>+<kbd>`</kbd>. Different levels of logging are available:
 
-No logging: 0
-Only log symbols identified by godefinfo: 1
-Log network calls: 2
-Log all debugging information: 3
+No logging: `0`
+
+Only log symbols identified by godefinfo: `1`
```
Comment (wholien) [inline README.md]: is `1` the default setting and thus in the snippet below?

File: README.md (hunk 3 above)
Comment (luttig) [inline README.md]: fixed in https://github.com/sourcegraph/sourcegraph-sublime/pull/17/commits/2723e0d7e66899c828c8c19433b1a24865bddd82

File: README.md (hunk 4 above)
Comment (luttig) [inline README.md]: 0 is default, but showed 1 here to demonstrate that you can change it. Do you think it should be 0 here? Presumably if you're setting it yourself, you don't want it to be 0.

File: sourcegraph_sublime.py [hunk 5]
```diff
@@ -67,6 +67,7 @@ def reload_settings():
 	load_settings(settings)
 	if SG_LIB_INSTANCE.settings.SG_BASE_URL != old_base_url and SG_LIB_INSTANCE.settings.AUTO_OPEN:
 		SG_LIB_INSTANCE.open_channel()
+		SG_LIB_INSTANCE.HAVE_OPENED_CHANNEL = True
```
Comment (wholien) [inline sourcegraph_sublime.py]: Should be in `open_channel()`

Comment (luttig) [inline sourcegraph_sublime.py]: fixed in https://github.com/sourcegraph/sourcegraph-sublime/commit/54aac2cad879feab4264ad430e746bf5a2fdb12b

**Comment by wholien** [comment]
LGTM

**Pull request merged**